policyengine_us/tests/policy/reform/winship.yaml

```diff
@@ -0,0 +1,17 @@
+- name: Winship reform gives a bonus
```

Comment:
Can we add two more tests to demonstrate this test's accuracy? Setting 20k/0 and 30k/0 employment income. This should then equal the sum of those two.

Reply:
Yep, added and it passes